client/state/jetpack/credentials/test/reducer.js

```diff
@@ -0,0 +1,46 @@
+import { markCredentialsAsInvalid, markCredentialsAsValid, testCredentials } from '../actions';
+import { testRequestStatus } from '../reducer';
+
+describe( 'reducer', () => {
+	describe( 'testRequestStatus', () => {
+		test( 'should return the initial state', () => {
+			expect( testRequestStatus( undefined, {} ) ).toEqual( {} );
+		} );
+
+		test( 'state should be valid if the credentials are okay', () => {
```

Comment:
A little nit on the test naming here: I believe `state` here is used for `status`, and `state` in the context of the reducer means something else, thus this could create confusion for someone reading the tests.

Reply:
Fixed!